app/ratelimit.go

```diff
@@ -23,9 +23,10 @@ type RateLimiter struct {
 	useAuth              bool
 	useIP                bool
 	header               string
+	trustedProxyIPHeader []string
 }
 
-func NewRateLimiter(settings *model.RateLimitSettings) (*RateLimiter, error) {
+func NewRateLimiter(settings *model.RateLimitSettings, trustedProxyIPHeader []string) (*RateLimiter, error) {
```

Comment:
It doesn't look like we actually save `trustedProxyIPHeader` to the constructed `RateLimiter`?

Reply:
Good catch, thanks!